.github/workflows/label-releases-doc-prs.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: "Label doc/releases PRs so our backport GitHub Action picks them up"
+
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
why should this be closed?

Reply:
`merged` is not an option.  Closed is a good thing in this context. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request